tests/test_models.py

```diff
@@ -1185,89 +1136,45 @@ def test_audit_param_in_program_enrollment_url(self, config_mock):
         enrollment_url = enterprise_catalog.get_program_enrollment_url(program_uuid=program_uuid)
         assert_url_contains_query_parameters(enrollment_url, {'audit': 'true'})
 
-    @mock.patch('enterprise.models.EnterpriseCustomerCatalog.contains_courses')
-    def test_get_course_and_course_run_no_content_items(self, contains_courses_mock):
-        """
-        The ``get_course_and_course_run`` method returns a tuple (None, None) when no content items exist.
-        """
-        contains_courses_mock.return_value = False
-        enterprise_customer_catalog = factories.EnterpriseCustomerCatalogFactory()
-        assert enterprise_customer_catalog.get_course_and_course_run('fake-course-run-id') == (None, None)
-
     @mock.patch('enterprise.models.EnterpriseCatalogApiClient', return_value=mock.MagicMock())
-    def test_get_course_and_course_run_no_content_items_waffle_sample(self, api_client_mock):
+    def test_get_course_and_course_run_no_content_items(self, api_client_mock):
         """
-        Verify the `get_course_and_course_run` method returns the same tuple with the enterprise catalog sample active.
+        Verify the `get_course_and_course_run` method returns the same tuple now that the waffle flag is removed.
```

Comment:
nit: I feel it might be unnecessary to mention that a waffle flag/sample was removed here.

For example, "same tuple now that the waffle flag is removed" in this docstring (and others) could be replaced by the original docstring for the removed test, e.g. "The ``get_course_and_course_run`` method returns a tuple (None, None) when no content items exist."